backend/gtime/gtime_test.go

```diff
@@ -22,7 +22,7 @@ func TestParseInterval(t *testing.T) {
 		{inp: "2w", duration: 2 * 168 * time.Hour},
 		{inp: "1M", duration: time.Duration(daysInMonth * 24 * int(time.Hour))},
 		{inp: "1y", duration: time.Duration(daysInYear * 24 * int(time.Hour))},
-		{inp: "invalid-duration", err: regexp.MustCompile(`^time: invalid duration "?invalid-duration"?$`)},
+		{inp: "invalid-duration", err: regexp.MustCompile(`time: invalid duration "?invalid-duration"?$`)},
```

Comment:
Are these changes unrelated?

Reply:
It is related. Because in stats_source package, for `ErrorWithErrorSource.Error()` method, we are prepending  error source: https://github.com/grafana/grafana-plugin-sdk-go/blob/main/experimental/status/status_source.go#L86C1-L88C2 

```
func (e errorWithSourceImpl) Error() string {
	return fmt.Errorf("%s error: %w", e.source, e.err).Error()
}
```

This is a change compared to error_source, where we only returned the error https://github.com/grafana/grafana-plugin-sdk-go/blob/main/experimental/errorsource/error_source.go#L22

```
// Error implements the interface
func (r Error) Error() string {
	return r.err.Error()
}
```

I could have changed it to `{inp: "invalid-duration", err: regexp.MustCompile(`^downstream error: time: invalid duration "?invalid-duration"?$`)}` as well. 

But also what do you think about prepending error and should we not do it? 🤔

Reply:
Ah 😄 I thought since the function didn't change the test shouldn't, but I didn't realize before it was a different path where it just showed the error.

>But also what do you think about prepending error and should we not do it? 🤔

If you think it's necessary. Otherwise I wonder will the logger catch it anyway and put the source as a label? 🤔

Reply:
Sure let's keep it simple for now and can always come back to improve 👍

Reply:
Fixed in https://github.com/grafana/grafana-plugin-sdk-go/pull/1150/commits/99766df734a2ea7a9ccea127c7baee0ab1427bb6